Lib/_pydatetime.py

```diff
@@ -1101,16 +1110,18 @@ def __format__(self, fmt):
             return self.strftime(fmt)
         return str(self)
 
-    def isoformat(self):
+    def isoformat(self, basic=False):
         """Return the date formatted according to ISO.
```

Comment:
```suggestion
        """Return the date formatted according to ISO 8601.
```

It just cuts off otherwise.